src/components/Cards/PaymentCard.jsx

```diff
@@ -17,13 +17,21 @@ export default function PaymentCard({
         }
     };
 
+    // Define background colors for Master and Visa cards
+    const cardColors = {
+        mastercard: "from-[#FF9100] to-[#F7353B]",
+        visa: "from-[#007FC1] to-[#6AAFDC]",
+    };
+
     const borderClasses = selected
         ? "border-8 border-Teal"
         : "border border-gray-300";
     return (
         <div
             onClick={handleCardClick}
-            className={`${borderClasses} w-[270px] h-[190px] md:w-[280px] md:h-[190px] lg:w-[380px] lg:h-[230px] bg-gradient-to-tr rounded-2xl flex content-center items-center justify-center from-[#9C2CF3] to-[#3A49F9]`}
+            className={`${borderClasses} w-[270px] h-[190px] md:w-[280px] md:h-[190px] lg:w-[380px] lg:h-[230px] bg-gradient-to-tr ${
```

Comment:
@Lilicod i think  w-[270px] and h-[190px] on small screens make the card too big on certain devices  .. better way is to use rem or something i guess